deeprank/learn/NeuralNet.py

```diff
@@ -766,7 +770,8 @@ def _train(self, index_train, index_valid, index_test,
         return torch.cat([param.data.view(-1)
                           for param in self.net.parameters()], 0)
 
-    def _epoch(self, data_loader, train_model):
+
+    def _epoch(self, data_loader, train_model, test_model=False):
```

Comment:
The name `test_model` is misleading. It's acutally a flag to tell whether there is `target` values or not in the data. So I suggest using e.g. `has_target=True`?
Also, please add the docstring for new parameters.

Reply:
Very good suggestion, thanks !